assets/js/init.js

```diff
@@ -1,4 +1,5 @@
-wikipediaPreview.init({
-	root: document.querySelector('.entry-content'),
-	detectLinks: true
-});
+// eslint-disable-next-line no-undef
+wikipediaPreview.init( {
```

Comment:
`wikipediaPreview` should be defined as a global in the eslint config

Reply:
make sense and done.